AgentDiscoveries-Backend/src/main/java/org/softwire/training/AgentDiscoveriesApplication.java

```diff
@@ -119,6 +119,7 @@ private void agentsRouteGroup() {
         post("", (req, res) -> agentsRoutes.createAgent(req, res), responseTransformer);
         get("/:id", (req, res) -> agentsRoutes.readAgent(req, res, idParamAsInt(req)), responseTransformer);
         put("/:id", (req, res) -> agentsRoutes.updateAgent(req, res, idParamAsInt(req)), responseTransformer);
+        put("/editcallsign/:id", (req, res) -> agentsRoutes.updateCallSign(req, res, idParamAsInt(req)), responseTransformer);
```

Comment:
This endpoint is inconsistent with the rest of the endpoints, I think "/callsign/:id" would be more suited